Tests/MongoSwiftTests/UnifiedTestRunner/UnifiedTestRunner.swift

```diff
@@ -156,17 +169,17 @@ struct UnifiedTestRunner {
                         let db = self.internalClient.anyClient.db(collData.databaseName)
                         let collOpts = MongoCollectionOptions(writeConcern: .majority)
                         let coll = db.collection(collData.collectionName, options: collOpts)
-                        try coll.drop()
+                        _ = try await coll.drop()
```

Comment:
since this method doesn't actually return anything I don't think you need the `_ =`